src/video/wayland/SDL_waylandwindow.c

```diff
@@ -890,7 +1074,20 @@ Wayland_SetWindowBordered(_THIS, SDL_Window * window, SDL_bool bordered)
 void
 Wayland_SetWindowResizable(_THIS, SDL_Window * window, SDL_bool resizable)
 {
-    CommitMinMaxDimensions(window);
+#ifdef HAVE_LIBDECOR_H
+    const SDL_WindowData *wind = window->driverdata;
+
+    if (wind->shell_surface.libdecor.frame) {
```

Comment:
This might be an issue if the shell surface is a union... but the null check is right at least. May just need to check the libdecor context in addition to this.